ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -192,6 +192,18 @@ DX2RC:
   implements:
   - CONTROL
 
+FRZM:
+  description: "Freezer monitoring."
+  is_abstract: true
+  opt_uses:
+   - failed_alarm
+   - run_status
+   - compressor_run_status
+   - run_time_accumulator
+  uses:
+   - temperature_sensor
```

Comment:
what temperature?

Reply:
This is a freezer used in kitchen. Will remove this canonical type from chiller and add it to the CRREF.yaml approved yesterday. The temperature is the freezer temperature.